task-sdk/src/airflow/sdk/definitions/edges.py

```diff
@@ -193,3 +194,115 @@ class EdgeInfoType(TypedDict):
     """Extra metadata that the DAG can store about an edge, usually generated from an EdgeModifier."""
 
     label: str | None
+
+
+def dag_edges(dag: DAG):
```

Comment:
I'd say it would make more sense if this stayed in the new common library that we are working on

Reply:
Since its used in structure APIs and some sdk tests

Reply:
Closed instead